ts/components/dialog/DeleteAccountModal.tsx

```diff
@@ -18,11 +18,22 @@ import { SessionRadioGroup } from '../basic/SessionRadioGroup';
 const deleteDbLocally = async () => {
   window?.log?.info('last message sent successfully. Deleting everything');
   window.persistStore?.purge();
+  window?.log?.info('store purged');
+
   await deleteAllLogs();
+  window?.log?.info('ladelete logs');
```

Comment:
do we still want this logging?

`ladelete logs` -> `delete all logs`

Reply:
updated